test/src/e2e_vm_tests/test_programs/should_pass/language/generic_with_other_generic/Forc.toml

```diff
@@ -0,0 +1,5 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+license = "Apache-2.0"
+name = "generic_with_other_generic"
+entry = "main.sw"
```

Comment:
Just to make sure things are fine, can you add a `core` dependency here? Ref: #1137 for reason

Reply:
Just for context @emilyaherbert, this will be a little less ambiguous when we have something like #1150.

Reply:
1150 is now in!

Reply:
👍